src/util/sql/transform-pay-decision.sql

```diff
@@ -160,26 +160,21 @@ CREATE TABLE ${migrationSchema:name}.evaka_fee_decision_child (
     sibling_discount INTEGER NOT NULL,
     placement_unit_id UUID REFERENCES ${migrationSchema:name}.evaka_daycare,
     service_need_option_id UUID,
-    base_fee INTEGER NOT NULL,
-    fee INTEGER NOT NULL,
-    final_fee INTEGER NOT NULL,
+    fee_and_deviations INTEGER[],
     effica_internal_decision_number INTEGER,
     effica_ssn TEXT
 );
 
 INSERT INTO ${migrationSchema:name}.evaka_fee_decision_child
-    (effica_paydecisionrows_guid, fee_decision_id, child_id, child_date_of_birth, sibling_discount, placement_unit_id, service_need_option_id, base_fee, fee, final_fee, effica_internal_decision_number, effica_ssn)
```

Comment:
Remove `effica_paydecisionrows_guid` column from the table (row 156) and update snapshots.